source/bot/index.ts

```diff
@@ -1,4 +1,5 @@
 import * as process from 'node:process';
+import { config as envconfig } from 'dotenv'; envconfig();
```

Comment:
I think naming it envconfig is a bit confusing. I would maybe reuse the name of the package `dotenv()` seems easier to get to me.

Reply:
Renamed it.